### PR TITLE
fb-add 4 subcommands-2

### LINKED ISSUE #2: créer 4 sous-commandes avec Picocli
* [x] new
* [x] clean
* [x] build
* [x] serve

### PR DESCRIPTION
**Pull request by yanik23**
closes #2

### CODE REVIEW

**Review by fabio10000 — CHANGES_REQUESTED** [review]
implement commands

**Comment by yanik23** [comment]
working on it

**Comment by yanik23** [comment]
Added the implementation of the subcommands. Please tell me if something is wrong.
Best regards !

Yanik

**Comment by yanik23** [comment]
Added the main that wasn't in the previous commits.

File: my-app/src/main/java/com/gen/app/Init.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.gen.app;
+
+
+import org.w3c.dom.ls.LSOutput;
```
Comment (fabio10000) [inline my-app/src/main/java/com/gen/app/Init.java:4]: unused import must be removed

**Review by fabio10000 — CHANGES_REQUESTED** [review]
There's an unused import that must be removed, for more details please have a look at my comment

**Comment by yanik23** [comment]
Removed the unused import, let me know if anything is still missing. Best regards.

**Review by fabio10000 — APPROVED** [review]

**Pull request merged**